Bug Reporting/ARKDefaultLogFormatter.h

```diff
@@ -18,7 +18,7 @@
 //  limitations under the License.
 //
 
-#import <Aardvark/ARKLogFormatter.h>
+#import "ARKLogFormatter.h"
```

Comment:
This is incorrect. Headers in frameworks should use bracket import syntax. Only `.m` files should use `""` syntax. Same goes for all the `.h` imports of other `<Aardvark/` changes below.